README.md

```diff
@@ -147,11 +147,11 @@ This repository contains many companion libraries to write integration tests wit
 | Wrap a single test to use a `withResource`-like function (`around`)                       | ✔️       | ✔️                                                           | ✖️                                                                |
 | Wrap a test group to use a `withResource`-like function (`aroundAll`)                     | ✔️       | ✔️                                                           | ✖️                                                                |
 | Randomising execution order                                                               | ✔️       | ✔️                                                           | C                                                                |
-| Randomised execution order by default                                                     | ✔️       | ✖️                                                           | C                                                                |
+| Randomised execution order by default                                                     | ✔️       | C                                                           | C                                                                |
```

Comment:
```haskell
-- test/SpecHook.hs
module SpecHook where

import Test.Hspec
import Test.Hspec.Runner
import Test.Hspec.Core.Spec

hook :: Spec -> Spec
hook = modifyConfig \ c -> c { configRandomize = True }
```

Reply:
That's still not by default then.
`C` isn't really an option for "by default" features.
But at least you can have a `C` for "Randomising execution order"

Reply:
That's OK with me, but then I think maybe at at least apply it uniformly, specifically:

---
![IMG_1661](https://github.com/user-attachments/assets/f22fe3e1-00b7-48c4-a20e-3966eda511fe)
---

Reply:
Jup agree!

Reply:
I would still argue that it's a substantial difference whether you have to pass `--randomize` on every test run (which some of your colleagues might not do and then accidentally introduce order dependencies) VS you can ensure in code that it always randomizes the execution order.

So I think `✖️` for the first case, and `C` for the second case kind of makes sense. Was this maybe what you had in mind when you originally put the `C` there?

That said, this is your comparison table, so I'm happy with whatever call you make.

Reply:
Aha that's a good point! That's probably indeed what I meant originally.
You can ignore these  objections then.